### DISCUSSION TITLE
How to reinstall apps or restore customizations done by the script

### DISCUSSION THREAD

**Discussion by mauamolat** [Raphire/Win11Debloat#96] [Q&A]
How do we restore windows apps or customizations done with the script?

**Comment by Raphire** [comment]
Heya!

I moved your issue to discussions as others might be interested in this aswell. As for your question:

### Reinstalling Apps

Almost all of the apps that this script can remove can be found & reinstalled through the Microsoft Store. Alternatively they can also be reinstalled via [WinGet](https://learn.microsoft.com/en-us/windows/package-manager/winget/). To my knowledge the only apps that cannot (easily) be reinstalled are the Microsoft Store itself and some Xbox framework.

### Reverting Changes

As for the customizations, most of these can be undone using the registry files that come with the script. All registry files are found in the `Regfiles` folder, inside that folder is another folder called `Undo`. This folder contains all the registry files to undo customizations. You can also view and download all the registry files for undoing customizations [here](https://github.com/Raphire/Win11Debloat/tree/master/Regfiles/Undo). Please note that some registry changes may not apply immediately, you may need to restart your PC or the Windows Explorer process.

Restoring your start menu pinned apps is a bit of a different process. These can be brought back by manually replacing the start menu file with the backup that the script creates. I outlined how to revert the start menu [in this guide](https://github.com/Raphire/Win11Debloat/discussions/114).

Hope this helps 🙃
